app/page.js

```
@@ -0,0 +1,130 @@
+              {errorMessage}
+            </div>
+          )}
+          
+          <button 
+            className="bg-blue-500 hover:bg-blue-600 text-white font-medium py-3 px-6 rounded-md transition-colors flex items-center justify-center"
+            onClick={handleCreateRoom}
+            disabled={isCreating}
+          >
+            {isCreating ? (
+              <>
+                <svg className="animate-spin h-5 w-5 text-white mr-2" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)